src/authextension.ts

```diff
@@ -2,9 +2,12 @@
 
 import { Apiendpoint } from './apiendpoint';
 import { stackAnalysisServices } from './stackAnalysisService';
+import { GlobalState } from './constants';
+import * as fetch from 'node-fetch'
 
 export module authextension {
   export let setContextData: any;
+  export let setUUID: any;
```

Comment:
Export the function directly instead of declaring like a variable. It obscures the readability.